build-image/first-boot.bash

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-REPOSITORYURL=$(grep -i '^repositoryurl' /etc/openhabian.conf | cut -d '=' -f2)
-CLONEBRANCH=$(grep -i '^clonebranch' /etc/openhabian.conf | cut -d '=' -f2)
+REPOSITORYURL=$(grep -i '^repositoryurl' /boot/openhabian.conf | cut -d '=' -f2)
+CLONEBRANCH=$(grep -i '^clonebranch' /boot/openhabian.conf | cut -d '=' -f2)
```

Comment:
Consider merging #683 first, then source init.bash

Reply:
683 has a few open points I don't know how to handle, especially introducing foreign code in out repo - would like to discuss this a bit more before merging.

/boot/openhabian.conf is copied over to /etc, but not yet when the code is executed in first-boot.bash.... if we want to use init.bash, we need to put it after this commands.

Reply:
> 683 has a few open points I don't know how to handle, especially introducing foreign code in out repo - would like to discuss this a bit more before merging.

ok. I don't understand what you refer to here (foreign code?) but let's discuss over at #683